### ISSUE TITLE
TypeError Crash

### ISSUE THREAD

**Issue by ask-compu** [reactiflux/discord-irc#486]
here's the console output, happens randomly, sometimes not for days, sometimes a few times a day

```
/usr/local/lib/node_modules/discord-irc/node_modules/winston/lib/winston/logger.js:237
      throw ex;
      ^

TypeError: Converting circular structure to JSON
    at JSON.stringify (<anonymous>)
    at Printf.formatter [as template] (/usr/local/lib/node_modules/discord-irc/dist/logger.js:14:32)
    at Printf.transform (/usr/local/lib/node_modules/discord-irc/node_modules/logform/printf.js:11:26)
    at Format.info [as transform] (/usr/local/lib/node_modules/discord-irc/node_modules/logform/combine.js:20:24)
    at DerivedLogger._transform (/usr/local/lib/node_modules/discord-irc/node_modules/winston/lib/winston/logger.js:235:29)
    at DerivedLogger.Transform._read (/usr/local/lib/node_modules/discord-irc/node_modules/readable-stream/lib/_stream_transform.js:184:10)
    at DerivedLogger.Transform._write (/usr/local/lib/node_modules/discord-irc/node_modules/readable-stream/lib/_stream_transform.js:172:83)
    at doWrite (/usr/local/lib/node_modules/discord-irc/node_modules/readable-stream/lib/_stream_writable.js:428:64)
    at writeOrBuffer (/usr/local/lib/node_modules/discord-irc/node_modules/readable-stream/lib/_stream_writable.js:417:5)
    at DerivedLogger.Writable.write (/usr/local/lib/node_modules/discord-irc/node_modules/readable-stream/lib/_stream_writable.js:334:11)
```

**Comment by Throne3d** [comment]
What version of discord-irc are you on? A very similar issue was fixed in #453, included in 2.7.0

**Comment by Throne3d** [comment]
I'm going to close this due to a lack of info. If it's still a problem, feel free to get back to us with more details!
